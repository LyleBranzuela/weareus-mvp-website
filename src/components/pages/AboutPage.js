import React from "react";
import TopContent from "../aboutpage-components/TopContent";
import ClientSection from "../aboutpage-components/ClientSection";
import TeamSection from "../aboutpage-components/TeamSection";
import CallToAction from "../homepage-components/CallToAction";
import { pageTransition } from "../../transitions/Transitions";
import { motion } from "framer-motion";

class AboutPage extends React.Component {
  render() {
    return (
      <motion.div
        intial="out"
        animate="in"
        exit="out"
        variants={pageTransition}
        className="aboutPage"
      >
        <TopContent />
        <ClientSection />
        <TeamSection />
        <CallToAction />
      </motion.div>
    );
  }
}

export default AboutPage;
