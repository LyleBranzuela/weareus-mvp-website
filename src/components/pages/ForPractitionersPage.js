import React from "react";
import PageHeader from "../homepage-components/PageHeader";
import FeaturesList from "../for-practitioner-components/FeaturesList";
import FeatureCarousel from "../for-practitioner-components/FeatureCarousel";
import PractitionerCTA from "../for-practitioner-components/PractitionerCTA";
import MessageOfTheDay from "../homepage-components/MessageOfTheDay";
import CallToAction from "../homepage-components/CallToAction";
import { pageTransition } from "../../transitions/Transitions";
import { motion } from "framer-motion";

class ForPractitionersPage extends React.Component {
  render() {
    return (
      <motion.div
        intial="out"
        animate="in"
        exit="out"
        variants={pageTransition}
        className="forPractitionersStyle"
      >
        <PageHeader />
        <MessageOfTheDay
          motd={
            <React.Fragment>
              We know the key to growing your business is to have a regular flow
              of clients <br /> through your door. But when most clients find
              health, wellness & self-
              <br />
              improvement businesses through word-of-mouth it can be hard to get
              the word <br /> out ot an abundance of clients who are looking for
              help online. We are Us
              <br /> provides you with a unique platform to attract a flow of
              new clients.
            </React.Fragment>
          }
        />
        <FeaturesList />
        <FeatureCarousel />
        <PractitionerCTA />
        <CallToAction />
      </motion.div>
    );
  }
}

export default ForPractitionersPage;
