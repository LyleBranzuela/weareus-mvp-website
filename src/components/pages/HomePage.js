import React from "react";
import PageHeader from "../homepage-components/PageHeader";
import MessageOfTheDay from "../homepage-components/MessageOfTheDay";
import CallToAction from "../homepage-components/CallToAction";

class HomePage extends React.Component {
  render() {
    return (
      <div className="homePage">
        <PageHeader />
        <MessageOfTheDay />
        <CallToAction />
      </div>
    );
  }
}

export default HomePage;
