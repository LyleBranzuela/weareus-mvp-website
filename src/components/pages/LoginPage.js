import React from "react";
import LoginForm from "../register-login-components/LoginForm";
import { pageTransition } from "../../transitions/Transitions";
import { motion } from "framer-motion";

const LoginPage = () => {
  return (
    <motion.div
      intial="out"
      animate="in"
      exit="out"
      variants={pageTransition}
    >
      <LoginForm />
    </motion.div>
  );
};

export default LoginPage;
