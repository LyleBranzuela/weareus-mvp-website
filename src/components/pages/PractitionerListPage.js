import React from "react";

class PractitionerListPage extends React.Component {
  render() {
    return <div className="practitionerList">Practitioners</div>;
  }
}

export default PractitionerListPage;
