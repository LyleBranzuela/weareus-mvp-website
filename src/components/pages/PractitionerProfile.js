import React from "react";
import CallToAction from "../homepage-components/CallToAction";
import ImageCarousel from "../practitionerprofile-components/ImageCarousel";
import ProfileInformation from "../practitionerprofile-components/ProfileInformation";
<<<<<<< HEAD
<<<<<<< HEAD
=======
<<<<<<< HEAD
>>>>>>> parent of e9faf8f... ..
import ContactCard from "../practitionerprofile-components/ContactCard";
import { pageTransition } from "../../transitions/Transitions";
=======
import { pageTransition } from "../App";
>>>>>>> parent of c592f96... Merge branch 'develop' into practitioner-profile
<<<<<<< HEAD
=======
=======
import { pageTransition } from "../App";
>>>>>>> parent of c592f96... Merge branch 'develop' into practitioner-profile
>>>>>>> parent of e9faf8f... ..
import { motion } from "framer-motion";

class PractitionerProfile extends React.Component {
  render() {
    return (
      <motion.div
        intial="out"
        animate="in"
        exit="out"
        variants={pageTransition}
        className="practitionerPage"
      >
        <ImageCarousel />
        <ContactCard />
        <ProfileInformation />
        <CallToAction />
      </motion.div>
    );
  }
}

export default PractitionerProfile;
