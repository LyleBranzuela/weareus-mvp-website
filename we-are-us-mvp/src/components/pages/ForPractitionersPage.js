import React from "react";
import NavigationBar2 from "../general-components/NavigationBar2";
import PageHeader from "../homepage-components/PageHeader";
import FeaturesList from "../for-practitioner-components/FeaturesList";
import FeatureCarousel from "../for-practitioner-components/FeatureCarousel";
import PractitionerCTA from "../for-practitioner-components/PractitionerCTA";
import MessageOfTheDay from "../homepage-components/MessageOfTheDay";
import CallToAction from "../homepage-components/CallToAction";

class ForPractitionersPage extends React.Component {
  render() {
    return (
      <div>
        <NavigationBar2 />
        <PageHeader />
        <MessageOfTheDay
          motd={
            <React.Fragment>
              We know the key to growing your business is to have a regular flow
              of clients <br /> through your door. But when most clients find
              health, wellness & self-
              <br />
              improvement businesses through word-of-mouth it can be hard to get
              the word <br /> out ot an abundance of clients who are looking for
              help online. We are Us
              <br /> provides you with a unique platform to attract a flow of
              new clients.
            </React.Fragment>
          }
        />
        <FeaturesList />
        <FeatureCarousel />
        <PractitionerCTA />
        <CallToAction />
      </div>
    );
  }
}

export default ForPractitionersPage;
