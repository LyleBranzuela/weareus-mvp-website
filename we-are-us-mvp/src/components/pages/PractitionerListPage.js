import React from "react";
import CallToAction from "../homepage-components/CallToAction";
import PractitionerList from "../practitioner-components/PractitionerList";
import api from "../../api/api";

class PractitionerListPage extends React.Component {
  constructor(props) {
    super(props);
    this.state = { practitioners: [] };

    // Prevents Memory Leaks
    this._isMounted = false;
  }

  // Function to Get All The Practitioners from the Server
  getAllPractitioners = async () => {
    const practitionerResponse = await api.get("/company");
    // Setting the Services and Regions States
    this._isMounted &&
      this.setState({
        practitioners: practitionerResponse.data.rows,
      });
  };

  componentDidMount() {
    this._isMounted = true;
    this._isMounted && this.getAllPractitioners();
  }

  componentWillUnmount() {
    this._isMounted = false;
  }

  render() {
    return (
      <div className="practitionerList mt-5">
        <PractitionerList
          practitioners={this.state.practitioners}
          showAll={true}
        />
        <CallToAction />
      </div>
    );
  }
}

export default PractitionerListPage;
