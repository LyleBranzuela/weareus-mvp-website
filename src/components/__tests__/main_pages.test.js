// Required packages for testing
import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

// Components to be tested
import AboutPage from '../pages/AboutPage.js';
import ContactUs from '../pages/ContactUsPage.js';
import ForPractitionersPage from '../pages/ContactUsPage.js';
import HomePage from '../pages/HomePage.js';
import LoginPage from '../pages/LoginPage.js';
import PractitionerListPage from '../pages/PractitionerListPage.js';
import PractitionerProfile from '../pages/PractitionerProfile.js';
// import RegisterPage from '../pages/RegisterPage.js';
// import SearchPage from '../pages/SearchPage.js';
// import TermsAndConditionsPage from '../pages/TermsAndConditionsPage.js';


// Snapshot tests for UI testing.
test('About Page Renders properly', () => {
    const tree = renderer.create(<Router><AboutPage /></Router>).toJSON();
    expect(tree).toMatchSnapshot();
});

test('Contact Renders properly', () => {
    const tree = renderer.create(<Router><ContactUs /></Router>).toJSON();
    expect(tree).toMatchSnapshot();
});

test('For-Practitioners Page Renders properly', () => {
    const tree = renderer.create(<Router><ForPractitionersPage /></Router>).toJSON();
    expect(tree).toMatchSnapshot();
});

// test('Home Page Renders properly', () => {
//     const tree = renderer.create(<Router><HomePage /></Router>).toJSON();
//     expect(tree).toMatchSnapshot();
// });

// test('Login Page Renders properly', () => {
//     const tree = renderer.create(<Router><LoginPage /></Router>).toJSON();
//     expect(tree).toMatchSnapshot();
// });

test('Practitioner List Page Renders properly', () => {
    const tree = renderer.create(<Router><PractitionerListPage /></Router>).toJSON();
    expect(tree).toMatchSnapshot();
});

test('Practitioner Profile Renders properly', () => {
    const tree = renderer.create(<Router><PractitionerProfile /></Router>).toJSON();
    expect(tree).toMatchSnapshot();
});

// test('Ts & Cs Page Renders properly', () => {
//     const tree = renderer.create(<Router><TermsAndConditionsPage /></Router>).toJSON();
//     expect(tree).toMatchSnapshot();
// });

// test('Register Page Renders properly', () => {
//     const tree = renderer.create(<Router><RegisterPage /></Router>, {createNodeMock}).toJSON();
//     expect(tree).toMatchSnapshot();
// });

// test('Search Page Renders properly', () => {
//     const tree = renderer.create(<Router><SearchPage /></Router>).toJSON();
//     expect(tree).toMatchSnapshot();
// });





