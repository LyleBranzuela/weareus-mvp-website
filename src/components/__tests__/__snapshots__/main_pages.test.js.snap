// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page Renders properly 1`] = `
<div>
  <div
    className="container-fluid"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h2
          id="aboutTitle"
        >
          About us
        </h2>
        <p
          id="about-paragraph"
        >
          
        </p>
        <hr
          style={
            Object {
              "marginTop": "100px",
              "width": "1300px",
            }
          }
        />
      </div>
    </div>
  </div>
  <div
    className="container-fluid"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <img
          alt="charis"
          id="mainFrame"
          src="http://54.66.121.53:1337"
        />
        <h5
          id="heavyHeader"
        >
          
        </h5>
        <p
          id="lightHeader"
        >
          
        </p>
        <p
          id="big-description"
        >
          
          <strong>
            <blockquote />
          </strong>
          <p
            id="clientMessage"
          >
            
          </p>
          <br />
          
          <br />
          <br />
        </p>
      </div>
    </div>
  </div>
  <div>
    <div
      className="bottom-padding teamSectionStyle container-margin-position container-fluid"
    >
      <div
        className="row"
      >
        <div
          className="col-sm-4"
          id="team-column-padding"
        >
          <img
            alt="about dan"
            id="emp-frame"
            src="http://54.66.121.53:1337"
          />
          <h4
            id="name"
          >
             
            
             
          </h4>
          <h4
            id="role"
          >
             
            
             
          </h4>
          <p
            id="team-description"
          >
            
          </p>
          <a
            href="https://designcabin.co.nz"
          >
            designcabin.co.nz
          </a>
        </div>
        <div
          className="col-sm-4"
          id="team-column-padding"
        >
          <img
            alt="about walter"
            id="emp-frame"
            src="http://54.66.121.53:1337"
          />
          <h4
            id="name"
          >
            
          </h4>
          <h4
            id="role"
          >
            
          </h4>
          <p
            id="team-description"
          >
            
          </p>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-4"
          id="team-column-padding"
        >
          <img
            alt="about dan"
            id="emp-frame"
            src="http://54.66.121.53:1337"
          />
          <h4
            id="name"
          >
            
          </h4>
          <h4
            id="role"
          >
            
          </h4>
          <p
            id="team-description"
          >
            
          </p>
          <a
            href="https://wordsforwellness.com"
          >
            wordsforwellness.com
          </a>
        </div>
        <div
          className="col-sm-4"
          id="team-column-padding"
        >
          <img
            alt="about dan"
            id="emp-frame"
            src="http://54.66.121.53:1337"
          />
          <h4
            id="name"
          >
            
          </h4>
          <h4
            id="role"
          >
            
          </h4>
          <p
            id="team-description"
          >
            
          </p>
          <a
            href="https://gardyneholt.co.nz"
          >
            gardyneholt.co.nz
          </a>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-4"
          id="team-column-padding"
        >
          <img
            alt="about dan"
            id="emp-frame"
            src="http://54.66.121.53:1337"
          />
          <h4
            id="name"
          >
            
          </h4>
          <h4
            id="role"
          >
            
          </h4>
          <p
            id="team-description"
          >
            
          </p>
          <a
            href="https://silverlane.agency"
          >
            silverlane.agency
          </a>
        </div>
      </div>
    </div>
  </div>
  <div>
    <div
      className="ctaStyle"
    >
      <div
        className="ctaStyle-text container"
      >
        <div
          className="row"
        >
          <h1
            id="ctaTitle"
          >
            
          </h1>
        </div>
        <div
          className="row"
        >
          <p
            id="ctaDesc"
          >
            
          </p>
        </div>
        <div
          className="row"
        >
          <a
            href="/register"
            onClick={[Function]}
          >
            <button
              className="primaryCustomButtonStyle btn btn-primary"
              disabled={false}
              id="ctaButton"
              type="button"
            >
              Join Us Now
            </button>
          </a>
        </div>
      </div>
    </div>
    <div
      id="signUpContainer"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-sm-6"
          >
            <p
              id="ctaSignUp"
            >
              Sign up for We are Us news and special offers.
            </p>
          </div>
          <div
            className="col-sm-6"
          >
            <div
              className="mb-3 input-group"
            >
              <input
                className="form-control"
                id="ctaSignUpForm"
                placeholder="Enter your email"
                type="text"
              />
              <div
                className="input-group-append"
              >
                <button
                  className="btn btn-light"
                  disabled={false}
                  id="ctaSignUpFormSubmit"
                  type="button"
                >
                  <img
                    alt="Submit Button"
                    src={
                      Object {
                        "ReactComponent": Object {
                          "$$typeof": Symbol(react.forward_ref),
                          "render": [Function],
                        },
                        "__esModule": true,
                        "default": "subscribe_enter_arrow.svg",
                      }
                    }
                  />
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Contact Renders properly 1`] = `
<div>
  <div
    className="contactUsPage"
  >
    <p
      className="contactCTA"
    >
       
      <strong>
         
        
      </strong>
       
    </p>
    <p
      className="contactText"
    >
       
      
    </p>
  </div>
</div>
`;

exports[`For-Practitioners Page Renders properly 1`] = `
<div>
  <div
    className="contactUsPage"
  >
    <p
      className="contactCTA"
    >
       
      <strong>
         
        
      </strong>
       
    </p>
    <p
      className="contactText"
    >
       
      
    </p>
  </div>
</div>
`;

exports[`Practitioner List Page Renders properly 1`] = `
<div
  className="practitionerList mt-5"
>
  <div
    className="pracContainerStyle container-fluid"
    style={
      Object {
        "backgroundColor": "white",
      }
    }
  >
    <div
      className="pracListStyle container"
    >
      <h4>
        Practitioners
      </h4>
      <div
        className="pracListDeckStyle card-deck"
      >
        <div
          className="row"
        />
      </div>
      <br />
    </div>
  </div>
  <div>
    <div
      className="ctaStyle"
    >
      <div
        className="ctaStyle-text container"
      >
        <div
          className="row"
        >
          <h1
            id="ctaTitle"
          >
            
          </h1>
        </div>
        <div
          className="row"
        >
          <p
            id="ctaDesc"
          >
            
          </p>
        </div>
        <div
          className="row"
        >
          <a
            href="/register"
            onClick={[Function]}
          >
            <button
              className="primaryCustomButtonStyle btn btn-primary"
              disabled={false}
              id="ctaButton"
              type="button"
            >
              Join Us Now
            </button>
          </a>
        </div>
      </div>
    </div>
    <div
      id="signUpContainer"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-sm-6"
          >
            <p
              id="ctaSignUp"
            >
              Sign up for We are Us news and special offers.
            </p>
          </div>
          <div
            className="col-sm-6"
          >
            <div
              className="mb-3 input-group"
            >
              <input
                className="form-control"
                id="ctaSignUpForm"
                placeholder="Enter your email"
                type="text"
              />
              <div
                className="input-group-append"
              >
                <button
                  className="btn btn-light"
                  disabled={false}
                  id="ctaSignUpFormSubmit"
                  type="button"
                >
                  <img
                    alt="Submit Button"
                    src={
                      Object {
                        "ReactComponent": Object {
                          "$$typeof": Symbol(react.forward_ref),
                          "render": [Function],
                        },
                        "__esModule": true,
                        "default": "subscribe_enter_arrow.svg",
                      }
                    }
                  />
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Practitioner Profile Renders properly 1`] = `
<div
  className="practitionerPage"
>
  <div>
    <div
      autoPlay={true}
      className="pracPersonalCarousel carousel slide"
      onKeyDown={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
    >
      <ol
        className="carousel-indicators"
      >
        <li
          className="active"
          onClick={[Function]}
        />
        <li
          onClick={[Function]}
        />
        <li
          onClick={[Function]}
        />
      </ol>
      <div
        className="carousel-inner"
      >
        <div
          className="active carousel-item"
        >
          <img
            alt="First slide"
            className="carousel-img-margin"
            src="prac_image_carousel_placeholder.jpg"
          />
        </div>
        <div
          className="carousel-item"
        >
          <img
            alt="second slide"
            className="carousel-img-margin"
            src="prac_image_carousel_placeholder.jpg"
          />
        </div>
        <div
          className="carousel-item"
        >
          <img
            alt="third slide"
            className="carousel-img-margin"
            src="prac_image_carousel_placeholder.jpg"
          />
        </div>
      </div>
      <a
        className="carousel-control-prev"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
      >
        <span
          aria-hidden="true"
          className="carousel-control-prev-icon"
        />
        <span
          className="sr-only"
        >
          Previous
        </span>
      </a>
      <a
        className="carousel-control-next"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
      >
        <span
          aria-hidden="true"
          className="carousel-control-next-icon"
        />
        <span
          className="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
  <div
    className="container-dimensions container-fluid"
  >
    <div
      className="row"
    >
      <div
        className="no-padding col-lg-8"
      >
        <img
          alt="practitioner-profile"
          id="profile-image-frame"
          src="prac_profile_placeholder.jpg"
        />
        <h2
          className="practitioner-title"
          id="practitioner-name"
        >
          
          <img
            alt="logo"
            className="practitioner-logo"
            src="prac_logo_placeholder.jpg"
          />
        </h2>
        <div
          id="category"
        >
          <span
            id="category-text"
          />
        </div>
        <div
          id="location"
        >
          <span
            id="suburb-profile"
          >
            
            ,
          </span>
          <span
            id="city-profile"
          >
             
            
          </span>
        </div>
        <br />
        <h5
          className="practitioner-name"
          id="person-name"
        >
           
        </h5>
      </div>
      <div
        className="contact-card-column col-lg-4"
      >
        <div
          className="contact-card-box card"
        >
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              Contact Details
            </div>
            <div
              className="card-subtitle h6"
            >
              
            </div>
            <p
              className="card-text"
            >
              
              <br />
              
              <span
                className="horizontal-line-card"
              />
              <a
                href="/#"
                id="purple-card-heading"
              >
                
                 (0)
                
                 
                
                 
                
              </a>
              <span
                className="horizontal-line-card"
              />
              <a
                href="/#"
                id="purple-card-heading"
              >
                
              </a>
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="no-padding col-lg-8"
      >
        <div
          className="practitioner-description-card card"
        >
          <div
            className="card-body"
          >
            <div
              className="card-title h5"
            >
              About
            </div>
            <p
              className="card-text"
            >
              <span
                className="inner"
                style={
                  Object {
                    "maxHeight": 500,
                  }
                }
              >
                <p
                  id="description-text"
                >
                  
                   
                </p>
              </span>
            </p>
          </div>
        </div>
        <div
          className="accordion-section container-fluid"
        >
          <div
            className="accordion"
            defaultationkey="0"
          >
            <div
              className="card"
            >
              <div
                className="card-header"
                onClick={[Function]}
              >
                Specialties
                <img
                  alt="accordion_arrow"
                  className="arrow-position"
                  src="accordion_arrow.svg"
                />
              </div>
              <div
                aria-expanded={null}
                className="collapse"
              >
                <div
                  className="card-body"
                >
                  <ul
                    className="accordion-content-list-style"
                  />
                </div>
              </div>
            </div>
            <div
              className="card"
            >
              <div
                className="card-header"
                onClick={[Function]}
              >
                Services
                <img
                  alt="accordion_arrow"
                  className="arrow-position"
                  src="accordion_arrow.svg"
                />
              </div>
              <div
                aria-expanded={null}
                className="collapse"
              >
                <div
                  className="card-body"
                >
                  <ul
                    className="accordion-content-list-style"
                    id="services"
                  />
                </div>
              </div>
            </div>
            <div
              className="card"
            >
              <div
                className="card-header"
                onClick={[Function]}
              >
                Training, Qualifications & Memberships
                <img
                  alt="accordion_arrow"
                  className="arrow-position"
                  src="accordion_arrow.svg"
                />
              </div>
              <div
                aria-expanded={null}
                className="collapse"
              >
                <div
                  className="card-body"
                >
                  <h6
                    className="title-header"
                  >
                    <img
                      alt="accordion_diploma"
                      src="icon_diploma.svg"
                      style={
                        Object {
                          "paddingRight": "5px",
                        }
                      }
                    />
                    Diplomas
                  </h6>
                  <div
                    className="font-body"
                    id="education"
                  >
                    <ul
                      className="training-accordion-body-font"
                    />
                  </div>
                  <br />
                  <h6
                    className="title-header"
                  >
                    <img
                      alt="accordion_certificate"
                      src="icon_certificate.svg"
                      style={
                        Object {
                          "paddingRight": "5px",
                        }
                      }
                    />
                    Certificates
                  </h6>
                  <ul
                    className="training-accordion-body-font"
                  />
                  <br />
                  <h6
                    className="title-header"
                  >
                    <img
                      alt="accordion_memberships"
                      src="icon_memberships.svg"
                      style={
                        Object {
                          "paddingRight": "5px",
                        }
                      }
                    />
                    Memberships
                  </h6>
                  <div
                    className="font-body"
                    id="memberships"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <div
      className="ctaStyle"
    >
      <div
        className="ctaStyle-text container"
      >
        <div
          className="row"
        >
          <h1
            id="ctaTitle"
          >
            
          </h1>
        </div>
        <div
          className="row"
        >
          <p
            id="ctaDesc"
          >
            
          </p>
        </div>
        <div
          className="row"
        >
          <a
            href="/register"
            onClick={[Function]}
          >
            <button
              className="primaryCustomButtonStyle btn btn-primary"
              disabled={false}
              id="ctaButton"
              type="button"
            >
              Join Us Now
            </button>
          </a>
        </div>
      </div>
    </div>
    <div
      id="signUpContainer"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col-sm-6"
          >
            <p
              id="ctaSignUp"
            >
              Sign up for We are Us news and special offers.
            </p>
          </div>
          <div
            className="col-sm-6"
          >
            <div
              className="mb-3 input-group"
            >
              <input
                className="form-control"
                id="ctaSignUpForm"
                placeholder="Enter your email"
                type="text"
              />
              <div
                className="input-group-append"
              >
                <button
                  className="btn btn-light"
                  disabled={false}
                  id="ctaSignUpFormSubmit"
                  type="button"
                >
                  <img
                    alt="Submit Button"
                    src={
                      Object {
                        "ReactComponent": Object {
                          "$$typeof": Symbol(react.forward_ref),
                          "render": [Function],
                        },
                        "__esModule": true,
                        "default": "subscribe_enter_arrow.svg",
                      }
                    }
                  />
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
